libp2p/pubsub/gossipsub.py

```diff
@@ -654,6 +704,14 @@ async def handle_graft(
 
         # Add peer to mesh for topic
         if topic in self.mesh:
+            for _peer in self.direct_peers:
```

Comment:
A `_`-prefixed variable means that it isn't used, but it is directly used in the next line.